src/convertToObject.js

```diff
@@ -6,7 +6,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const listArrays = sourceString
+    .split(';')
+    .filter((item) => item.length)
+    .map((element) => element.split(':').map((item) => item.trim()));
```

Comment:
Although the chaining of methods here is correct, the variable naming could be improved for better readability. Instead of `listArrays`, which is a bit generic and does not convey the purpose of the content, consider using a name that describes what is stored in the variable, such as `styleDeclarations`.

Reply:
ok